1.Exploratory_Data_Analysis/scripts/0.cell_count_analysis.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+get_ipython().run_line_magic("matplotlib", "inline")
+import itertools
+import pathlib
+
+import pyarrow.parquet as pq
+from sklearn.cluster import KMeans
+from sklearn.manifold import TSNE
+
+sys.path.append("..")
+from utils.utils import df_stats, plot_features_all_cluster
+
+# In[2]:
+
+
+celltype = "SHSY5Y"
+
+
+# In[3]:
+
+
+# Parameters
+celltype = "PBMC"
```

Comment:
Did you mean to define `celltype` twice here? This seems like unnecessary code.

Reply:
I did not mean to, but I am removing it now